src/components/month/calendarMonthView.component.ts

```diff
@@ -239,18 +243,24 @@ export class CalendarMonthViewComponent implements OnChanges, OnInit, OnDestroy
     this.eventTimesChanged.emit({event, newStart, newEnd});
   }
 
+  public count(): number {
+    return 7 - this.excludeDays.length;
+  }
```

Comment:
how about using `days.length`? for the `MonthView` a property `daysInWeek` would might also be useful ?!